source/_integrations/sighthound.markdown

```diff
@@ -63,3 +63,47 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+To verify the integration, check if new entities appear as `camera.sighthound` and `image_processing.sighthound_camera_my_cam`
```

Comment:
If `camera.sighthound` is created this is new behaviour to me

Reply:
Fixed. It turned out to be a misconfiguration on my side.